lib/xenops_utils.ml

```diff
@@ -27,6 +27,23 @@ module Unix = struct
 
   let file_descr_of_rpc x = x |> Rpc.int_of_rpc |> file_descr_of_int
   let rpc_of_file_descr x = x |> int_of_file_descr |> Rpc.rpc_of_int
+
+  (** [rm path] removes a file or directory recursively without following
```

Comment:
Should this go with https://github.com/xapi-project/xenopsd/pull/476/files#diff-3db2c635c01fe0fbacdaa66d2fe541e1R396 ?